exporter/jaeger/jaegergrpcexporter/exporter.go

```diff
@@ -73,8 +76,14 @@ func (s *protoGRPCSender) pushTraceData(
 		return len(td.Spans), consumererror.Permanent(err)
 	}
 
+	octx := context.Background()
```

Comment:
not from your change but it should be using the `ctx` passed to `pushTraceData` instead.

Reply:
Modified to use the passed in context.